src/pool/srv_pool.c

```diff
@@ -3015,17 +3015,19 @@ ds_pool_svc_get_prop(uuid_t pool_uuid, d_rank_list_t *ranks,
 }
 
 int
-ds_pool_reintegrate(uuid_t pool_uuid, d_rank_list_t *ranks,
-		uint32_t reint_rank, struct pool_target_id_list *reint_list)
+ds_pool_target_update_state(uuid_t pool_uuid, d_rank_list_t *ranks,
+		uint32_t rank, struct pool_target_id_list *target_list,
+		pool_comp_state_t state)
 {
-	int				rc;
-	struct rsvc_client		client;
-	crt_endpoint_t			ep;
+	int							rc;
+	struct rsvc_client			client;
+	crt_endpoint_t				ep;
```

Comment:
Looks like the indentation got a little wild in this function. I'm OK if you want to correct it in a follow-on PR but it should be fixed.

Reply:
There will be another followup patch. I can make sure to fix it in there.